src/components/team-logo/index.tsx

```diff
@@ -0,0 +1,23 @@
+import type { QwikIntrinsicElements } from "@builder.io/qwik";
+import { component$ } from "@builder.io/qwik";
+
+const TEAM_CITY_MAP = {
+  "Atlanta Hustle": "ATL",
+};
+
+export type TeamLogoProps = Omit<
+  QwikIntrinsicElements["img"],
+  "src" | "alt"
+> & { team: keyof typeof TEAM_CITY_MAP };
```

Comment:
So here is where I use that same prop shadowing technique, except there's two props i _don't_ want people to pass in: `src` and `alt`. We come up with those in the component (using the `team` prop), so we `Omit` them. Typescript has a bunch of utility types like that that allow you to avoid writing out the whole type.

the `&` just means "this type contains everything from `Omit<QwikIntrinsicElements["img"], "src" | "alt">` and ALSO everything from `{ team: keyof typeof TEAM_CITY_MAP };`

`keyof` means "anything that is a key (as opposed to a value) of this type", and "typeof" means "the type of this object".

so `keyof typeof TEAM_CITY_MAP` means "anything that is a key of the type that TEAM_CITY_MAP is", which in this case is the string `"Atlanta Hustle"`.